src/components/ScrollTo.js

```diff
@@ -58,6 +58,17 @@ export class ScrollTo extends Component {
     return this.scrollToRef.current;
   }
 
+  /**
+   * The container provided in the containersRef dome structure in which scrolling
+   * should happen.
+  */
+  scrollabelContainer() {
+    const { containerRef } = this.props;
+
+    if (!containerRef || !containerRef.current || !containerRef.current.domEl) return null;
+    return containerRef.current.domEl.getElementsByClassName('mirador-scrollto-scrollable')[0];
```

Comment:
Unfortunately we need to reach into the dom here. We need to understand both the:  component that is sized only in view `containerRef` and the scrollable portion of this on which we need to scroll against.